src/backend/enums/environments.ts

```diff
@@ -4,3 +4,8 @@ export enum ENV {
   STAGING = 'staging',
   PRODUCTION = 'production',
 }
+
+export enum REDIS_PREFIX {
+  SESSION = 'GovSession__',
```

Comment:
Nitpick: the name 'GovSession' implies to me that it's something to do with GOV.UK, but in fact it's GovGraph's own session. Maybe `Session__` or `GovSearchSession__`?

Reply:
Good shout, it'll be `Session__`